swagger.yaml

```diff
@@ -236,20 +236,11 @@ definitions:
         type: string
       sourceType:
         type: string
-        description: > 
+        description: >
           Type of data source being accessed directly. Possible values are "api", "database", or "other".
       otherSourceType:
         type: string
         description: "Explanation if sourceType is 'other'"
-      database:
```

Comment:
The changes look good. The only extra thing that I had in my notes was that we were going to rename database into DataSource so that we could specify things such as: Banner or ODProd. Does that match with what you remember?